standards.md

```diff
@@ -0,0 +1,315 @@
+# Earth System Metrics and Diagnostics Standards
+
+## Author list
+Authors are presented in order of last name:  
+
+Chih-Chieh Chen, National Center for Atmospheric Research  
+Dani Coleman, National Center for Atmospheric Research  
+Tom Jackson, NOAA Geophysical Fluid Dynamics Laboratory   
+John Krasting, NOAA Geophysical Fluid Dynamics Laboratory  
+Yi-Hung Kuo, University of California Los Angeles  
+Jessica Liptak, NOAA Geophysical Fluid Dynamics Laboratory  
+David Neelin, University of California Los Angeles  
+Ana Ordonez, Lawrence Livermore National Laboratory   
+Aparna Radhakrishnan, Princeton University/NOAA Geophysical Fluid Dynamics Laboratory  
+Paul Ullrich, University of California Davis   
+
+
+## Table of Contents
+1. [Best Practices](#bestpractices)
+2. [DEM Framework File Format](#demframework)
+    1. [Settings file](#settingsfile)
+    2. [Contents file](#contentsfile)
+3. [Runtime Environment](#runtimeenvironment)
+4. [Common Output Bundle Format](#commonoutput)
+5. [Observational Data Format](#obsdataformat)
+6. [Observational Data Server](#obsdataserver)
+7. [Common Metric Output Format](#commonmetric)
+    1. [DIMENSIONS](#dimensions)
+    2. [RESULTS](#results)
+
+
+## Best Practices <a name="bestpractices"></a>
+
+### Nomenclature 
+In this document we refer to [MDTF](https://mdtf-diagnostics.readthedocs.io/en/latest/index.html) process-oriented diagnostics (PODs) and [CMEC](https://cmec.llnl.gov) metric kernels (MKs) as data evaluation modules (DEMs).
+
+### Supported Languages
+Python is the preferred language for development. However, there are certain cases where Python may not be performant or may not have the requisite functionality (i.e. certain statistical analysis routines). Nonetheless, if the kernel evaluation routines are not in python, python interfaces should be provided for accessing the functionality of the kernel.  
```

Comment:
Python is advantageous because it is portable, flexible, and unrestricted.  The MDTF's goal is to avoid license dependent software.  Is this really an EMDS suggestion or is this more reflective of MDTF and CMEC?

Reply:
This is interesting because CMEC uses shell script interfaces rather than python interfaces, actually (except for running MDTF PODs). The shell scripts usually activate conda environments and point to one or more Python scripts. 
I thought there was some language about avoiding license-restricted software, but I'm not seeing it now. I would say that new modules should be written in a freely-available language that has active maintenance, and Python is one such language.

Reply:
I am still unclear if EMDS should be prescriptive in terms of languages.  Theoretically, EMDS is about the interface between an analysis routine and a framework, either of which could be in any language, right?

Reply:
@jkrasting Paul and I worked on some new language for this section, which I've committed.